drag_and_drop_v2/public/js/drag_and_drop.js

```diff
@@ -212,9 +212,14 @@ function DragAndDropTemplates(configuration) {
             gettext('Items placed here: ') + items_in_zone.map(function (item) { return item.displayName; }).join(", ")
           );
         }
+
+        /* Force zone alignment if showing answer. */
+        if (ctx.show_answer && zone.align === 'none') {
+          zone.align = 'center'
+        }
+
         if (zone.align !== 'none') {
           item_wrapper += '.item-align.item-align-' + zone.align;
-          //items_in_zone = $.grep(ctx.items, is_item_in_zone);
```

Comment:
I'm not a fan of leaving code commented out.  It's why we have SCM, after all.  Should I really leave it in?